packages/expo-clipboard/src/web/ClipboardModule.ts

```diff
@@ -103,8 +103,18 @@ export default {
           throw new CopyFailureException(e.message);
         }
       }
-      default:
-        return this.setString(text);
+      default: {
+        try {
+          if (!navigator.clipboard) {
+            throw new ClipboardUnavailableException();
+          }
+          await navigator.clipboard.writeText(text);
+          return true;
+        } catch {
+          // we can fall back to legacy behavior in any kind of failure
+          return this.setString(text);
+        }
+      }
```

Comment:
I guess `ClipboardUnavailableException` should be propagated to the caller of this function right? 
```suggestion
        try {
          if (!navigator.clipboard) {
            throw new ClipboardUnavailableException(); // <- throwing that
          }
          await navigator.clipboard.writeText(text);
          return true;
        } catch { // <- would go here, is it intentional? 
          // we can fall back to legacy behavior in any kind of failure
          return this.setString(text);
        }
      }
```

Reply:
it is intentional - I can either throw or early return `this.setString`. In any kind of failure (including `navigator.clipboard` unavailability) I'd like to quietly use the legacy method. This way it is not breaking in any way.

Reply:
Maybe add a little bit more of clarification to this catch statement then?